src/shm/pool/multi.rs

```diff
@@ -78,16 +78,33 @@ use wayland_client::{
 
 use super::raw::RawPool;
 
+#[derive(Debug, thiserror::Error)]
+pub enum PoolError {
+    InUse,
```

Comment:
You'll need to add `#[error = "this is what will be displayed"]` attributes to each enum variant